test/src/test/java/org/corfudb/runtime/collections/SMRMultiLogunitTest.java

```diff
@@ -0,0 +1,216 @@
+package org.corfudb.runtime.collections;
```

Comment:
Clean Up test code: variable names, dont print to system.out, remove commented code,